### PR TITLE
Adding a plugin to calculate the virtual temperature

### PR DESCRIPTION
**Pull request by mo-philrelton**
Address: https://metoffice.atlassian.net/browse/EPPT-1965

This PR adds a plugin to calculate the virtual temperature based on the air temperature and humidity mixing ratio.

No CLI added due to the move to the DAG runner.

Testing:
 - [x] Ran tests and they passed OK
 - [x] Added new tests for the new feature(s)

CLA
 - [x] If a new developer, signed up to CLA

### CODE REVIEW

**Review by Katie-Howard — CHANGES_REQUESTED** [review]
I've looked through the changes and I'm generally happy with them:

- The virtual temperature plugin looks like it does what it's supposed to
- The unit tests that have written look sensible and all run fine
- The PR is failing due to the PR standards - I think it's just something to do with adding yourself as a contributor. I think it's because in the email address you are phil.relton@metoffice.gov.uk but you have put your name as Philip it doesn't like this. It might be worth changing that and see if it then passes that test?
- I've left a couple of comments about your comments but they are very minor.

File: improver/virtual_temperature.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+from typing import Union
+
+from iris.cube import Cube, CubeList
+
+from improver import BasePlugin
+from improver.utilities.common_input_handle import as_cubelist
+
+
+class VirtualTemperature(BasePlugin):
+    """Calculates the virtual temperature from temperature and ."""
```
[the remark this answers is not in the document]
Comment (mo-philrelton) [reply improver/virtual_temperature.py]: I did, but I have now changed this string to `"""Plugin class to handle virtual temperature calculations."""` to better answer your second comment.

File: improver/virtual_temperature.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    @staticmethod
+    def get_virtual_temperature(temperature: Cube, humidity_mixing_ratio: Cube) -> Cube:
+        """
+        Calculate the virtual temperature from temperature and humidity mixing ratio.
```
[the remark this answers is not in the document]
Comment (mo-philrelton) [reply improver/virtual_temperature.py:21]: You're right it's a little repetitive. I have changed the class docstring.

**Comment by mo-philrelton** [comment]
The cause of the failing tests was a poorly set up git config. Rather than confusing this PR with further commits and rewriting the history to the new configuration I will close this PR and open a new one.

**Pull request closed without merging**